File: ООП/Homework2/animals_PY/classes/wilds/tiger.py
from .wild import Wild

class tiger(Wild):
    def __init__(self, name, height, weight, coloreye, place, date):
        super(tiger, self).__init__(name, height, weight, coloreye, place, date)

    def voice(self):
       print(f"{super().voice()} РрРрР")

File: ООП/Homework5/Зоопарк/all_animals/wilds/tiger.java
package Зоопарк.all_animals.wilds;

public class tiger extends wild{

    public tiger(double height, double weight, String eyeColor, String date, String place) {
        super(height, weight, eyeColor, date, place);
    }

    public String sound() {
        return "РррРР";
    }
    public String toString() {
        return String.format("Tiger (%s)", super.toString());
    }
}
